src/expressions/if-expr.md

```diff
@@ -90,27 +89,6 @@ let a = if let Some(1) = x {
 assert_eq!(a, 3);
 ```
 
-An `if let` expression is equivalent to a [`match` expression] as follows:
-
-<!-- ignore: expansion example -->
-```rust,ignore
-if let PATS = EXPR {
-    /* body */
-} else {
-    /*else */
-}
-```
-
-is equivalent to
-
-<!-- ignore: expansion example -->
-```rust,ignore
-match EXPR {
-    PATS => { /* body */ },
-    _ => { /* else */ },    // () if there is no else
-}
-```
-
```

Comment:
It hasn't been the case for quite some time-> https://github.com/rust-lang/rust/pull/80357

Reply:
My mistake, code desugaring only changed for `while` expressions. `for` expressions stay the same